js_modules/dagit/packages/core/src/ui/Filters/FilterDropdown.tsx

```diff
@@ -261,6 +349,45 @@ const TextInputWrapper = styled.div`
   }
 `;
 
+type FilterDropdownMenuItemProps = React.ComponentProps<typeof MenuItem> & {
+  menuKey: string;
+  index: number;
+};
+const FilterDropdownMenuItem = React.memo(
+  ({menuKey, index, ...rest}: FilterDropdownMenuItemProps) => {
+    const divRef = React.useRef<HTMLDivElement | null>(null);
+    const isFocused = useRecoilValue(
+      isFocusedSelector(React.useMemo(() => ({key: menuKey, index}), [index, menuKey])),
+    );
+    React.useLayoutEffect(() => {
+      if (isFocused) {
+        divRef.current?.querySelector('a')?.focus();
+      }
+    }, [isFocused]);
```

Comment:
Aria roles can/should be used for the "active" item state in a combobox -- you'll want to avoid taking DOM focus from the input while it's there. 

https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/

Using aria roles and the default menu item hover style, I think there's a good chance you don't need the `FilterDropdownMenuItemProps` wrapper component.

Reply:
I was looking at that page the other day and didn't find an example that matched this specific case. All of the examples on that page assume that the user can only make a single selection. In this case they can sometimes make more than one selection.

Reply:
Okay there this example: https://ariakit.org/examples/combobox-multiple
This makes sense. Though our example won't work the same because I'm using MenuItem, which means tabbing will put you into the list.

Reply:
That fact about MenuItem^ is why I resorted to managing focus this way because otherwise it's weird that `tab` will move you in the list but looking at the example again I see they used negative tabIndex to disable that behavior so let me try that